packages/stories/src/stories/definition-list.ts

```diff
@@ -0,0 +1,283 @@
+import { bindTemplate, Parameters, TemplateFn } from '@core';
+
+export interface Definition {
+  term: string;
+  descriptions: string[];
+}
+
+export interface DefinitionListArgs {
+  modifier?: string;
+  definitions: Definition[];
+}
+
+export interface DefinitionListTemplateFn<TemplateFnReturnType> extends TemplateFn<DefinitionListArgs, TemplateFnReturnType> {
+}
+
+export interface DefinitionListParameters<TemplateFnReturnType> extends Parameters<DefinitionListArgs, TemplateFnReturnType> {
+}
+
+const definitions: Definition[] = [
+  {
+    term: 'Thema:',
+    descriptions: [
+      'Wonen',
+      'Werken'
+    ]
+  },
+  {
+    term: 'Regelkwalificatie:',
+    descriptions: [
+      'Instructieregels'
+    ]
+  },
+  {
+    term: 'Onderwerp:',
+    descriptions: [
+      'Bedrijventerrein'
+    ]
+  },
+  {
+    term: 'Functie:',
+    descriptions: [
+      'Bedrijventerrein > Bedrijventerrein'
+    ]
+  },
+  {
+    term: 'Type regel:',
+    descriptions: [
+      'Bindend voor bestuursorganen'
+    ]
+  },
+  {
+    term: 'Normadressaat:',
+    descriptions: [
+      '<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>',
+      '<p>Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>'
+    ]
+  },
+  {
+    term: 'Pneumonoultramicroscopicsilicovolcanoconiosisfantasismus:',
+    descriptions: [
+      '<p>Een factitious (quasi-fictieve) vorm van pneumoconiose (stoflongen), veroorzaakt door het inademen van microscopisch kleine silicadeeltjes uit vulkanen.</p>'
+    ]
+  },
+  {
+    term: 'De stoflongziekte (mijnwerkers-pneumoconiose):',
+    descriptions: [
+      '<p>Veroorzaakt door lange blootstelling aan kolengruis.</p>'
+    ]
+  },
+  {
+    term: 'Pneumoconiose:',
+    descriptions: [
+      '<p>Pneumoconiose of stoflong is een longaandoening die door de inhalatie van stof wordt veroorzaakt.</p>'
+    ]
+  }
+];
+
+const columnDefinitions: Definition[] = [
+  {
+    term: 'Verzoek 1:',
+    descriptions: [
+      'Aanleg rondweg Leiden traject Rijnland route 1'
+    ]
+  },
+  {
+    term: 'Verzoeknummer:',
+    descriptions: [
+      '20181024 00729 000'
+    ]
+  },
+  {
+    term: 'Datum ingediend:',
+    descriptions: [
+      '24/10/2018'
+    ]
+  },
+  {
+    term: 'Ingediend bij:',
+    descriptions: [
+      'Gemeente Den Haag'
+    ]
+  },
+  {
+    term: 'Uw verzoek wordt behandeld door:',
+    descriptions: [
+      'Omgevingsdienst Haaglanden'
+    ]
+  }
+];
+
+const smallContentDefinitions: Definition[] = [
+  {
+    term: 'Organisator:',
+    descriptions: [
+      'Gemeente Rotterdam'
+    ]
+  },
+  {
+    term: 'Verzoeknummer:',
+    descriptions: [
+      '12123497987'
+    ]
+  },
+  {
+    term: 'Status:',
+    descriptions: [
+      'Open'
+    ]
+  },
+  {
+    term: 'Creatie datum:',
```

Comment:
Creatiedatum is één woord, als je toch de code weer open hebt.. kan je 'm meenemen?